client.go

```diff
@@ -127,7 +130,7 @@ func Follow() (<-chan Conn, func(), error) {
 
 func readMsgs(s int, cb func(Conn)) error {
 	for {
-		rb := make([]byte, syscall.Getpagesize()) // TODO: re-use
+		rb := make([]byte, 2*syscall.Getpagesize()) // TODO: re-use
```

Comment:
Why is that?

Reply:
I looked at the strace of `conntrack -E` and `ulogd` and that was the buffer size they used. Not sure why, otherwise.